### PR TITLE
reorder scripts & remove localbiblio

### LINKED ISSUE #1373: Preview link feature is broken
The preview link feature for Pull Requests seems to be broken.

I receive the following error on my latest pull request:

```
💥 Error: 504 Gateway Time-out 💥

PR Preview failed to build. (Last tried on Apr 14, 2020, 2:06 PM UTC).
More

PR Preview relies on a number of web services to run. There seems to be an issue with the following one:

🚨 Spec Generator - Spec Generator is the web service used to build specs that rely on ReSpec.

🔗 Related URL

<html><body><h1>504 Gateway Time-out</h1>
The server didn't respond in time.
</body></html>

If you don't have enough information above to solve the error by yourself (or to understand to which web service the error is related to, if any), please file an issue.

```

### PR DESCRIPTION
**Pull request by jnurthen**
Should resolve #1373

***

### :boom: Error: 504 Gateway Time-out :boom: ###

[PR Preview](https://github.com/tobie/pr-preview#pr-preview) failed to build. _(Last tried on Apr 15, 2020, 5:39 AM UTC)_.

<details>
<summary>More</summary>

PR Preview relies on a number of web services to run. There seems to be an issue with the following one:

:rotating_light: [Spec Generator](https://www.w3.org/2015/labs/) - Spec Generator is the web service used to build specs that rely on ReSpec.

:link: [Related URL](https://labs.w3.org/spec-generator/?type=respec&url=https%3A%2F%2Frawcdn.githack.com%2Fw3c%2Faria-practices%2Fc74e8e46b38c7df5f4c1079f8d3dedf5373af1ab%2Faria-practices.html%3FisPreview%3Dtrue)

```
<html><body><h1>504 Gateway Time-out</h1>
The server didn't respond in time.
</body></html>

```

_If you don't have enough information above to solve the error by yourself (or to understand to which web service the error is related to, if any), please [file an issue](https://github.com/tobie/pr-preview/issues/new?title=Error%20not%20surfaced%20properly&body=See%20w3c/aria-practices%231374.)._
</details>

### CODE REVIEW

File: aria-practices.html [hunk 1]
```diff
@@ -26,7 +26,7 @@
       of the
       <a href="https://www.w3.org/WAI/">Web Accessibility Initiative</a>.
       It supports the
-      <a href="" class="specref">WAI-ARIA 1.2</a> [<cite><a href="#bib-WAI-ARIA" class="bibref">WAI-ARIA</a></cite>]
+      <a href="" class="specref">WAI-ARIA 1.2</a> [<cite>[[WAI-ARIA]]</a></cite>]
```
Comment (carmacleod) [inline aria-practices.html:29]:
```suggestion
      <a href="" class="specref">WAI-ARIA 1.2</a> [<cite>[[WAI-ARIA]]</cite>]
```

**Comment by jnurthen** [comment]
lets wait until spec-generator is working until we merge

**Comment by jnurthen** [comment]
https://github.com/w3c/spec-generator/issues/292

**Comment by jnurthen** [comment]
closed in favor of #1376

**Pull request closed without merging**